avalon/tools/models.py

```diff
@@ -232,12 +235,13 @@ def set_assets(self, asset_ids=[], asset_entities=None):
         assets = list()
         if asset_entities is not None:
             assets = asset_entities
-        elif asset_ids:
+        else:
```

Comment:
--- OLD CODE - START ---
This part was changed and has reason for change please put back